.github/vale/dicts/aiven.dic

```diff
@@ -1,4 +1,4 @@
-170
+171
```

Comment:
We can remove this number entirely. It was originally used to keep a track of the number of lines in the file.